### PR TITLE
[ADMIN] Add Consultation Report admin

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Admin/ConsultationReportAdmin.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        '_sort_by' => 'position',
+    ];
+
+    protected function configureShowFields(ShowMapper $showMapper)
+    {
+        $showMapper
+            ->add('name', null, [
+                'label' => 'Nom',
+            ])
+            ->add('url', UrlType::class, [
```
Comment (ottaviano) [inline src/Admin/ConsultationReportAdmin.php]: maybe you wanted to use UrlType on `configureFormFields` instead?

File: src/Admin/ConsultationReportAdmin.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                'label' => 'URL',
+            ])
+            ->add('position', null, [
+                'label' => 'Position',
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
```
Comment (ottaviano) [inline src/Admin/ConsultationReportAdmin.php]: I think we can remove this phpdoc

File: src/Admin/ConsultationReportAdmin.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            ->add('url', null, [
+                'label' => 'URL',
+                'show_filter' => true,
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
```
Comment (ottaviano) [inline src/Admin/ConsultationReportAdmin.php]: same

File: src/Admin/ConsultationReportAdmin.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/Admin/ConsultationReportAdmin.php]: Thanks! Done.

File: src/Admin/ConsultationReportAdmin.php (hunk 3 above)
Comment (illusionOfParadise) [reply src/Admin/ConsultationReportAdmin.php]: Done.

File: app/config/services/admin.xml [hunk 4]
```diff
@@ -664,6 +664,18 @@
             </call>
         </service>
 
+        <!--
+        ######################################################
+        # Ideas Workshop
+        ######################################################
+         -->
+        <service id="app.admin.consultation" class="AppBundle\Admin\ConsultationReportAdmin">
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply app/config/services/admin.xml]: Done.

**Pull request merged**